create_portal_teaching_dataset.py

```diff
@@ -13,9 +20,23 @@
                inplace=True)
 surveys.replace({'species_id': {'NA': 'NL'}, 'sex': {'P': float('nan'), 'R': float('nan'), 'Z': float('nan')}}, inplace=True)
 
+# correct wrong date, pd.to_datetime throughs an error when combining year, month, day to a date
+# in the year 2000 there is a 31st April and a 31st of September, 
+# perhaps these should be 1st of May and 1st of October, resp.
+# to check:
+# pd.to_datetime(surveys[['year', 'month', 'day']])
+# replace wrong dates:
+surveys.loc[(surveys.year==2000) & (surveys.month==4) & (surveys.day==31),'day'] = 1
+surveys.loc[(surveys.year==2000) & (surveys.month==4) & (surveys.day==1),'month'] = 5
+surveys.loc[(surveys.year==2000) & (surveys.month==9) & (surveys.day==31), 'day'] = 1
+surveys.loc[(surveys.year==2000) & (surveys.month==9) & (surveys.day==1), 'month'] = 10
 
 # Clean Species Table
-species = pd.read_csv("http://wiki.ecologicaldata.org/sites/default/files/portal_species.txt",
+# old url:
+# url = "http://wiki.ecologicaldata.org/sites/default/files/portal_species.txt"
+# current url (for species and plot table):
```

Comment:
I don't think we need the three commented lines here. Let's just fix the url and note the fix in the commit message.